example/main.tf

```diff
@@ -47,6 +47,8 @@ module "aurora" {
   aurora_allowed_cidr_blocks            = [data.aws_vpc.vpc.cidr_block]
   performance_insights_enabled          = true
   performance_insights_retention_period = 7
+  performance_insights_kms_key_id       = data.aws_kms_alias.aurora_cluster_kms_alias.target_key_arn
```

Comment:
use kms arn instead of alias

Reply:
This is fetching the kms arn only @vijay-stephen

Reply:
I got it , i will change the naming convention of this data lookup